src/Exercise/SplitSecondStopwatch.elm

```diff
@@ -0,0 +1,93 @@
+module Exercise.SplitSecondStopwatch exposing (forbidStopwatchExportAll, forbidStopwatchTypeAlias, ruleConfig)
+
+import Comment exposing (Comment, CommentType(..))
+import Dict
+import Elm.Syntax.Declaration exposing (Declaration(..))
+import Elm.Syntax.Exposing as Exposing exposing (TopLevelExpose(..))
+import Elm.Syntax.Module exposing (Module(..))
+import Elm.Syntax.Node as Node exposing (Node(..))
+import List.Extra
+import Review.Rule exposing (Error, Rule)
+import RuleConfig exposing (AnalyzerRule(..), RuleConfig)
+
+
+ruleConfig : RuleConfig
+ruleConfig =
+    { restrictToFiles = Just [ "src/SplitSecondStopwatch.elm" ]
+    , rules =
+        [ CustomRule forbidStopwatchTypeAlias
+            (Comment "elm.split-second-stopwatch.no_stopwatch_type_alias" Essential Dict.empty)
+        , CustomRule forbidStopwatchExportAll
+            (Comment "elm.split-second-stopwatch.no_stopwatch_export_all" Essential Dict.empty)
+        ]
+    }
+
+
+forbidStopwatchTypeAlias : Comment -> Rule
+forbidStopwatchTypeAlias comment =
+    Review.Rule.newModuleRuleSchema comment.path ()
+        |> Review.Rule.withSimpleDeclarationVisitor (hasStopwatchTypeAliasVisitor comment)
+        |> Review.Rule.fromModuleRuleSchema
+
+
+hasStopwatchTypeAliasVisitor : Comment -> Node Declaration -> List (Error {})
```

Comment:
Might we want to make helper functions for these for future rules?

Reply:
Let's do it when we have a second use case for it